envi/common.py

```diff
@@ -1,18 +1,25 @@
 import logging
 
 EMULOG = 9
-LOG_LEVELS = ('EMULOG', 'DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL')
+LOG_LEVELS = (
+        'CRITICAL',
+        'ERROR',
+        'WARNING',
+        'INFO',
+        'DEBUG',
+        'EMULOG',
+        )
 LOG_FORMAT = '%(asctime)s:%(levelname)s:%(name)s: %(message)s' \
              '[%(filename)s:%(funcName)s:%(lineno)s:%(threadName)s]'
 
+logging.basicConfig(level=logging.WARNING, format=LOG_FORMAT)
+logging.addLevelName(EMULOG, 'EMULOG')
 
 def setLogging(logger, level=None):
-    if level:
+    if level is not None:
```

Comment:
Any particular reason for this?

Reply:
no.  just feels like more explicit is better here.